content/posts/android-tv-13.md

```diff
@@ -0,0 +1,92 @@
+
+## Playback changes
+
+Most changes in this release are related to playing video files, the so called playback code. A lot of changes were made, both bugfixes and new features! The full list is, as always, available in the changelog. The most notable changes are listed below.
+
+## Transcoding updates
+
+A lot of issues with video are caused by the way the app transcodes video. We've tweaked the transcoding behavior and found a lot of issues were fixed with those tweaks, issues like the video ending early or unexpectedly rewinding itself. Unfortunately these changes triggered some issues in Jellyfin 10.7 and we had to limit these changes to servers running Jellyfin 10.8. We recommend updating as soon as a stable server release is ready.
+
+Additionally, we've added support for more audio codecs. The added codecs include Dolby Digital, Digital Plus and TrueHD, as well as DTS and Linear PCM audio, all commonly found on DVDs and Blu-rays, but also a more universal AAC support. This change is comparable with the audio codec changes in the last release of our mobile app.
+
+### Version selection
+
+A much requested feature was the ability to add a version selector. We've now added this function. A new button is now shown on the details page for a movie/episode/video that can be used to select which version to play.
+
+![Screenshot of the new version selection](/images/posts/androidtv-0-13-0/versionselection.png)
+
+### Speed control
+
+It's now possible to change the speed of playback. You can slow down or speed up a video. The set speed is remembered so the next video plays at the same speed, until you restart the app or manually change it back. Speed control is not available for Live TV.
+
+![Screenshot of the new speed controls](/images/posts/androidtv-0-13-0/speedcontrol.png)
+
+### Subtitle preferences
+
+Not everyone enjoyed the default <span style="background:#000;color:#fff;padding:4px;">subtitle styling</span> in the previous version, so now there are a few options to change it. It's now possible to change the size of the subtitles and to toggle between a black background or text outline. These preferences work for all text based subtitles. Image based subtitles like SubStation Alpha and VobSub still use their own styling.
+
+## Search & Screen saver
+
+You can now use the search function on Android TV to search in your Jellyfin libraries. This feature is supported for Android TV devices that do not use Google TV (like the new Chromecast). To get started, just search for a movie and press the "Available on Jellyfin" button.
+
+Additionally, we've added a screen saver that highlights your media. To enable it go to your system settings, open the screen saver category, and select the Jellyfin screensaver. On some devices this function may be called "Daydream". The screensaver shows random series and movies with their backdrop.
```

Comment:
Not consistent between "screen saver" and "screensaver" in this paragraph (and the heading).

Reply:
Should be "screen saver". Fixed it.